ecosystem/sep-0030.md

```diff
@@ -144,8 +144,14 @@ either. The two issuers are:
 - `SEP-10`: A [SEP-10] Web Auth server for proving high threshold control of an
 account. The [SEP-10] server should be configured to only issue [JWT]s if signers
 prove they meet the high threshold on the account.
-- `External`: Defined by the implementer. Proves control of a phone number,
-email address, or other identity.
+- `External`: Defined by the implementer. Proves possession of an identity that
+  is external to the Stellar network such as a phone number, email address, or
+  any other electronic or physical identity. Any forms of identity may be
+  supported. Authentication may be immediate and synchronous, as is the case with
+  electronic identities such as phone, email, and many OpenID Connect
+  providers. Authentication may be delayed and asynchronous, as might be the
```

Comment:
What makes email confirmation or SMS verification synchronous? In fact, these are known to be asynchronous forms of communication compared to a phone call or F2F conversation. The user can wait as long as he/she pleases to confirm control of a phone number or email address unless the sender of the confirmation request limits the window in which the code/link sent is valid.

I think the difference you're pointing out is how long it takes the authentication provider to verify the identity based on the method used. So verifying that a user lives at a certain address or verifying that a user's passport is authentic may take more time than SMS/email verification _starting from when the user provides the code/clicks the link/sends the document_.

I think it would be more clear to remove the distinction and just list the potential forms of identify verification. An authentication provider reading this SEP will likely just pick the one easiest for them given their circumstances. They won't be asking themselves "should I do this synchronously or asynchronously"?

Reply:
Ah great distinction, thank you. I'm addressing this feedback in https://github.com/stellar/stellar-protocol/pull/770.